panel/widgets/input.py

```diff
@@ -1168,7 +1169,7 @@ class LiteralInput(Widget):
         'value': """JSON.stringify(value).replace(/,/g, ",").replace(/:/g, ": ")"""
     }
 
-    _widget_type: ClassVar[type[Model]] = _BkTextInput
+    _widget_type: ClassVar[Type[Model]] = _BkTextInput  # noqa
```

Comment:
```suggestion
    _widget_type: ClassVar[type[Model]] = _BkTextInput 
```

Also, need to remove `Type` import